tutorials/install-psql-tutorial.md

```diff
@@ -0,0 +1,151 @@
+# Tutorial: How to install psql on Mac, Ubuntu, Debian, Windows
+
+### Introduction
+`psql` is the standard command line interface for interacting with a PostgreSQL 
+or TimescaleDB instance. Here we explain how to install `psql` on various platforms.
+
+### Before you start
+Before you start, you should confirm that you don’t already have `psql` installed. 
+In fact, if you’ve ever installed Postgres or TimescaleDB before, you likely already 
+have `psql` installed.
+
+```bash
+psql --version
+```
+
+### Install on MacOS using Homebrew
+First, install the [Brew Package Manager][brew-package-manager]. Homebrew simplifies
+the installation of software on macOS.
+
+Second, update `brew`. From your command line, run the following commands:
+
+```bash
+brew doctor
+brew update
+brew install libpq
+```
+
+Finally, create a symbolic link to `psql` (and other `libpq` tools) into `/usr/local/bin`
+so that you can reach it from any command on the macOS Terminal.
+
+```bash
+brew link --force libpq ail
+```
+
+### Install on Ubuntu 16.04,18.04 and Debian 9,10
+Install on Ubuntu and Debian using the `apt` package manager:
+
+```bash
+sudo apt-get update
+sudo apt-get install postgresql-client
+```
+
+>:TIP: This only installs the `psql` client and not the PostgreSQL database.
+
+### Install on Windows 10
+We recommend using the installer from [PostgreSQL.org][windows-installer].
+
+### Last step: Connect to your PostgreSQL server
+Let’s confirm that `psql` is installed:
+
+```bash
+psql --version
+```
+
+Now, in order to connect to your PostgreSQL server, we’ll need the following 
+connection parameters:
+- Hostname
+- Port
+- Username
+- Password
+- Database name
+
+For example, if you're using [Timescale Cloud][timescale-cloud-install], you can 
+navigate to the ‘Overview Tab’ of your dashboard and locate your `host`, `port`, 
+and `password`, as highlighted below.
+
+<img class="main-content__illustration" src="https://s3.amazonaws.com/docs.timescale.com/hello-timescale/NYC_figure1_1.png" alt="NYC Taxis"/>
+
+There are two ways to use these parameters to connect to your PostgreSQL database.
+
+#### Option 1: Supply parameters at the command line
+In this method, you'd use parameter flags on the command line to supply the required
+information to connect to a PostgreSQL database: 
+
+```bash
+psql -h [HOSTNAME] -p [PORT] -U [USERNAME] -W -d [DATABASENAME]
+```
+
+Once you run that command, the prompt will ask you for your password. (This is the purpose 
+of the `-W` flag.)
+
+#### Option 2: Use a service URI
+The Service URI often begins with `postgres://`. You can see an example of a Service URI
+in the [Timescale Cloud][timescale-cloud-install] dashboard above.
```

Comment:
It's a little jarring to have this link to the Timescale Cloud install page.  The instinct is to think this would send you to the image of the portal above.  Also, all internal links don't open a new window, so it jumps you away from the tutorial page.